### PR TITLE
fix!: adapt to breaking upstream changes

### PR DESCRIPTION
**Pull request by clason**
Problem: The "trick" of wrapping an anonymous node into a single `choice` or `seq` no longer works as of https://github.com/tree-sitter/tree-sitter/pull/2577  since the wrapped anonymous nodes are no longer exposed to queries.

Solution: Drop unnecessary `choice` and `skip` and use named nodes in queries instead (anonymous nodes are no longer exposed).

@amaanq

### CODE REVIEW

**Comment by ahelwer** [comment]
Thanks! Do I need to propagate these query changes to nvim-treesitter?

File: queries/highlights.scm [hunk 1]
```diff
@@ -85,27 +85,27 @@
   "call"
   "define"
   "either"
-  "else" 
+  "else"
```
[the remark this answers is not in the document]
Comment (ahelwer) [reply queries/highlights.scm:88]: That's okay, I don't like trailing whitespace either.

**Comment by clason** [comment]
No, thank you, I can handle this myself (since I know what changes are needed)!

**Comment by clason** [comment]
Ah, I guess I missed some hidden queries ;)

**Comment by clason** [comment]
Ran our query formatter on the nvim queries, as a treat ;)

**Pull request merged**